cookbooks/aws-parallelcluster-environment/test/controls/efs_spec.rb

```diff
@@ -4,11 +4,42 @@
 
   only_if { !os_properties.redhat_ubi? }
 
+  describe file('/kitchen/test/sources_dir/efs-utils-1.34.1.tar.gz') do
```

Comment:
The folder `/kitchen/test/sources_dir` seems incorrect. You should change it to: `#{node['cluster']['sources_dir']}`